projects/spectator/src/lib/spectator-host/spectator-host.ts

```diff
@@ -57,7 +54,7 @@ export class SpectatorHost<C, H = HostComponent> extends Spectator<C> {
 
   public setHostInput<K extends keyof H>(input: Partial<H>): void;
   public setHostInput<K extends keyof H>(input: K, inputValue: H[K]): void;
-  public setHostInput<K extends keyof H>(input: Partial<H> | K, value?: H[K]): void {
+  public setHostInput(input: any, value?: any): void {
```

Comment:
There is no type safety in `setHostInput` anymore?

Reply:
When using overloads, the actual function implementation signature does not need to be typed,  at least not for the outside world. As long as it is compatible with the overloads. The typings in the implementation signature are only used for inside the implementation.

Reply:
Yes, I know. Sorry, missed the overloads :)

Reply:
I made the signature simplier because it would otherwise be harder to match the improved signature of setProps 😄